src/classes/i18n.js

```diff
@@ -25,6 +26,7 @@ export default class i18n {
         this.loaded = false;
 
         this.config = new Config();
+        this.locales_folder = '';
```

Comment:
Why don't we init here already?

Reply:
configure is called for proper initialization. Will update this to the default _locales folder